protocol/0042-LIQF-setting_fees_and_rewarding_lps.md

```diff
@@ -245,6 +242,7 @@ Using only information from the prior block as the seed may allow exploits based
 
 - At the end of the epoch, calculate the actual observed epoch length `observed_epoch_length` = the difference in nanoseconds between the Vega time at the start of the epoch and the Vega time at the end of the epoch.
 
+Note that because vega time won't be progressing intra-block the above mechanism should ensure that `s_i` gets incremented only if the LP was meeting their commitment at every point this was checked within the block. 
```

Comment:
What happens if during the block there are three checks for an LP who was meeting their commitment previously:

check 1: still meeting commitment
check 2: not meeting 
check 3: meeting again 

So they weren’t meeting it in every check but were only not meeting for zero time. I guess that’s ok despite not being true per the above?

Reply:
the more I think about it the more I convince myself (very possibly wrongly) that it's actually better to check once at the end of the block and take their lowest commitment for the block as the commitment to check against.

Reply:
If we just did the checks once per block but using their lowest supplied liquidity at any point during the block, that would be perfect.